product_pricelist_supplierinfo/models/product_template.py

```diff
@@ -48,7 +48,7 @@ def _get_supplierinfo_pricelist_price(
             if seller and seller.product_uom != price_uom:
                 price = seller.product_uom._compute_price(price, price_uom)
 
-            convert_to_price_uom = lambda price: self.uom_id._compute_price(
+            convert_to_price_uom = lambda price: self.uom_id._compute_price(  # noqa
```

Comment:
Why this `noqa` here? Can you explain a little what is the concrete problem?
IMHO, if the problem is not fixable, it is better to specify the error you are ignoring, rather than ignoring everything.